lib/crypt-port.h

```diff
@@ -470,6 +471,10 @@ make_failure_token (const char *setting, char *output, int size);
    because it's used as a C string in a few places.  */
 extern const unsigned char ascii64[65];
 
+/* The table with invalid salt characters used for inital checking
+   of salt strings.  */
+extern const unsigned char badsalt_chars[166];
```

Comment:
Typo: "inital". I don't think you gain anything by including the NUL explicitly and using hard-coded array size (and I didn't verify the 166 is right). I'd just use `[]` here and let the compiler add the NUL.

Reply:
Addressed in rebased commit.